### PR TITLE
Unescape HTML - as per style guide

### PR DESCRIPTION
**Pull request by edent**
This patch turns `&quot;` into `"` and `&dagger;` into `†` etc.

Makes the source easier to read and search.  Complies with #1193

### CODE REVIEW

**Comment by edent** [comment]
This has passed CI - I don't know why it is saying it hasn't https://travis-ci.org/w3c/html/builds/348892330?utm_source=github_status&utm_medium=notification

File: sections/editing.include [hunk 1]
```diff
@@ -324,8 +324,8 @@
     <img src="images/focus-tree.png" alt="" width="800" height="450" />
 
     If the widget with <a>focus</a> in this example was the text field in the dialog box, then key
-    input would be channeled from the graphical system to &#x2460; the Web browser, then to &#x2461;
-    the tab, then to &#x2462; the dialog, and finally to &#x2463; the text field.
+    input would be channeled from the graphical system to ① the Web browser, then to ②
```
Comment (chaals) [inline sections/editing.include:327]: Why not make these a normal ordered list?

**Review by chaals — CHANGES_REQUESTED** [review]
Can we turn a list into a list? Other than that, LGTM - feel free to make the change or explain why not, then merge it yourself

File: sections/editing.include (hunk 1 above)
Comment (edent) [reply sections/editing.include:327]: Because in translation, I assume the ol-li could become 一,二,三,四. Whereas the image has circled numbers on it.

Comment (chaals) [reply sections/editing.include:327]: Fair enough, thanks.

**Pull request merged**